app/backend/src/clj/teet/environment.clj

```diff
@@ -128,7 +128,8 @@
    :eelis {:wms-url (->ssm [:eelis :wms-url] nil)}
    :email {:from (->ssm [:email :from] nil)
            :subject-prefix (->ssm [:email :subject-prefix] nil)
-           :contact-address (->ssm [:email :contact-address] nil)}
+           :contact-address (->ssm [:email :contact-address] nil)
+           :server (->ssm [:email :server] nil)}
```

Comment:
We want to parse the string value in SSM to a Clojure map, so the parse-fn should be `read-string` instead of `nil`